linera-views/src/log_view.rs

```diff
@@ -62,7 +62,7 @@ impl<C, T> View<C> for LogView<C, T>
 where
     C: Context + Send + Sync,
     ViewError: From<C::Error>,
-    T: Send + Sync + Serialize,
+    T: Clone + Send + Sync + Serialize,
```

Comment:
Technically, this restricts the library API hence the SDK. Should we create a trait `ShareableView` so that the requirement `T: Clone` is only added for the new method? (Or at least create a task?)